.codecov.yml

```diff
@@ -1,7 +1,16 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
 coverage:
   status:
     project:
       default:
-        threshold: 0.1%
-    patch: false
-    changes: false
+        threshold: 1%
+    patch:
+      default:
+        target: 90%
```

Comment:
90% looks pretty strict to me. We can use this value for now and see how it will affect MCS decommission and other changes related to migration/updation things.